src/db/index.ts

```diff
@@ -0,0 +1,6 @@
+import { drizzle } from "drizzle-orm/node-postgres";
+import * as schema from "./schema.ts";
+import dotenv from "dotenv";
+
+dotenv.config({ path: ".env" });
```

Comment:
it is nice to use env validators in the app - it should help to avoid issues with not installed variable  on different environments